flexbox/src/main/java/com/google/android/flexbox/FlexboxLayoutManager.java

```diff
@@ -1848,4 +1979,63 @@ public String toString() {
                     '}';
         }
     }
+
+    private static class SavedState implements Parcelable {
+
+        private int mAnchorPosition;
+
+        private int mAnchorOffset;
+
+        @Override
+        public int describeContents() {
+            return 0;
+        }
+
+        @Override
+        public void writeToParcel(Parcel dest, int flags) {
+            dest.writeInt(this.mAnchorPosition);
+            dest.writeInt(this.mAnchorOffset);
+        }
+
+        SavedState() {
+        }
+
+        private SavedState(Parcel in) {
+            this.mAnchorPosition = in.readInt();
+            this.mAnchorOffset = in.readInt();
+        }
+
+        private SavedState(SavedState savedState) {
+            mAnchorOffset = savedState.mAnchorOffset;
```

Comment:
I would reorder these to match with other methods, such as writeParcel(Parcel, int), SavedState(Parcel).